dspace-server-webapp/src/main/java/org/dspace/app/rest/repository/ItemRestRepository.java

```diff
@@ -95,6 +96,9 @@ public class ItemRestRepository extends DSpaceObjectRestRepository<Item, ItemRes
     @Autowired
     private UriListHandlerService uriListHandlerService;
 
+    @Autowired
+    DuplicateDetectionService duplicateDetectionService;
```

Comment:
Unless I'm missing something, I think you can remove this? It doesn't appear to be used as you removed everything else from ItemRestRepository.  (But please check to see if I'm correct)

Reply:
correct, another oversight not picked up in my self-review, fixed.